src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -411,10 +410,7 @@ public function handleError(int $type, string $message, string $file, int $line)
 
         $logMessage = $this->levels[$type].': '.$message;
 
-        if (null !== self::$toStringException) {
```

Comment:
Maybe I'm missing something, but I haven't found a reason why this variable should have any other value than `null`.

Reply:
that's a leftover, it can be cleaned up

Reply:
Indeed. The variable was used as a workaround on PHP 7.2/7.3 and became obsolete when we removed compat code on 6.0.